.cruft.json

```
@@ -0,0 +1,25 @@
+{
+  "template": "https://github.com/appuio/commodore-component-template.git",
+  "commit": "3e1ac242633b3345621ba9f2b72f8dfea8e6ac4a",
+  "checkout": "main",
+  "context": {
+    "cookiecutter": {
+      "name": "Openshift4 Networking",
+      "slug": "openshift4-networking",
+      "parameter_key": "openshift4_networking",
+      "test_cases": "defaults syn-monitoring",
+      "add_lib": "n",
+      "add_pp": "n",
+      "add_golden": "y",
+      "add_matrix": "y",
+      "add_go_unit": "n",
+      "copyright_holder": "VSHN AG <info@vshn.ch>",
+      "copyright_year": "2023",
```

Please don't randomly change the copyright year

```suggestion
      "copyright_year": "2022",
```